api/api.go

```diff
@@ -657,13 +661,19 @@ func (api *api) SetNextBackupReminder(backupReminderRequest *BackupReminderReque
 
 var startMutex sync.Mutex
 
-func (api *api) Start(startRequest *StartRequest) error {
+func (api *api) Start(startRequest *StartRequest) (err error) {
+	api.startupError = nil
+	defer func() {
```

Comment:
I think this is not a clean way to do it with this defer and tracking setting the error variable everywhere.